yaksh/forms.py

```diff
@@ -276,3 +276,40 @@ class CourseForm(forms.ModelForm):
     class Meta:
         model = Course
         fields = ['name', 'active', 'enrollment']
+
+class EditProfile(forms.Form):
```

Comment:
you can use Profile model forms, this will make saving and editing easy for you.